index.js

```diff
@@ -22,7 +22,16 @@ function safeStringifyReplacer(seen) {
 	};
 }
 
-export default function safeStringify(object, {indentation} = {}) {
+export default function safeStringify(object, {indentation, customReplacer} = {}) {
 	const seen = new WeakSet();
-	return JSON.stringify(object, safeStringifyReplacer(seen), indentation);
+	let preReplacer = (k, v) => v;
+	if (customReplacer) {
+		preReplacer = customReplacer;
+	}
+
+	return JSON.stringify(
+		object,
+		(key, value) => safeStringifyReplacer(seen)(key, preReplacer.call(object, key, value)),
```

Comment:
Wouldn't it be simpler to pass the custom replaces into `safeStringifyReplacer` and call it there?

Reply:
Perhaps, but `safeStringifyReplacer` is a function that works correctly. I didn't want to change its signature nor implementation, not knowing were if it is used anywhere else — it is not, I know.